keyboards/citrus/erdnuss65/info.json

```diff
@@ -25,81 +30,79 @@
         "rgblight": true,
         "audio": false
     },
-    "ws2812": {
-        "pin": "A15"
-    },
     "rgblight": {
+        "pin": "A15",
```

Comment:
This needs to be reverted. rgblight.pin is no longer valid.

Reply:
> This needs to be reverted. rgblight.pin is no longer valid.

What does that mean? I'm not sure what to do

Reply:
https://docs.qmk.fm/#/ChangeLog/20230528 read the top section here